PYME/recipes/html_recipe_editor.py

```diff
@@ -4,9 +4,18 @@
 from tornado.ioloop import IOLoop
 from .traits import HasTraits, Int, Str
 from jigna.web_app import WebApp
-from jigna.template import Template
+#from jigna.template import Template
+from jigna.vue_template import VueTemplate
+
+import jigna.web_server
+# monkey-patch the js file location
+# The reason we do this is so we can use a more recent version of vuejs. By default, jigna ships with a jigna-vue.js
+# which contains a concatenation of jquery, vue=1.x, and jigna. In recipe_editor.html, we load a modified version of
+# jigna-vue.js, jigna-vue-bare WITHOUT vue which lets us separately load a recent version of vue
+jigna.web_server.JIGNA_JS_FILE='/Users/david/dev/jigna/jigna/js/dist/jigna.js'
```

Comment:
was going to suggest just merging, as you mention being a bit agile on this one, but this bit looks not too portable.

What's the status of the current webby editor with the parameters separated from the svg render? Is that a 5 min pop-into clusterUI to take care of the friction points of #473 while we work on this more intuitive/flashier one?